allows the fifth demoness (succubus) class
in old mission games for the expansion pack

diff -urNp uhexen2-src-20060330/hexen2/menu.c uhexen2-src-20060402/hexen2/menu.c
--- uhexen2-src-20060330/hexen2/menu.c	2006-04-03 11:27:26.000000000 +0300
+++ uhexen2-src-20060402/hexen2/menu.c	2006-04-03 11:28:24.000000000 +0300
@@ -925,7 +925,7 @@ static int	m_class_cursor;
 
 // change the define below to 0 if you want to allow the
 // demoness class  in old mission through the menu system
-#define DISALLOW_DEMONESS_IN_OLD_GAME	1
+#define DISALLOW_DEMONESS_IN_OLD_GAME	0
 
 static void M_Class_Draw (void)
 {
